release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -524,14 +533,20 @@ In the following tables, features are marked with the following statuses:
 ==== Linux Control Groups version 1 is now deprecated
 
 In {op-system-base-full} 9, the default mode is cgroup v2. When {op-system-base-full} 10 is released, systemd will not support booting in the cgroup v1 mode and only cgroup v2 mode will be available. As such, cgroup v1 is deprecated in {product-title} 4.16 and later. cgroup v1 will be removed in a future {product-title} release.
- 
+
 [id="ocp-4-16-deprecation-samples-operator"]
 ==== Cluster Samples Operator
 
-The Cluster Samples Operator is deprecated with the {product-title} 4.16 release. The Cluster Samples Operator will stop managing and providing support to the non-S2I samples (image streams and templates). No new templates, samples or non-Source-to-Image (Non-S2I) image streams will be added to the Cluster Samples Operator. However, the existing S2I builder image streams and templates will continue to receive updates until the Cluster Samples Operator is removed in a future release. 
+The Cluster Samples Operator is deprecated with the {product-title} 4.16 release. The Cluster Samples Operator will stop managing and providing support to the non-S2I samples (image streams and templates). No new templates, samples or non-Source-to-Image (Non-S2I) image streams will be added to the Cluster Samples Operator. However, the existing S2I builder image streams and templates will continue to receive updates until the Cluster Samples Operator is removed in a future release.
```

Comment:
Note to reviewers: This is just my IDE cleaning up whitespace errors that were previously merged. I didn't change this content.